### PR TITLE
Replace Piwik with Matomo

### LINKED ISSUE #71: Console command to download UI screenshot still points to piwik
While printing the console command the repository points to piwik and gives 500 while executing.

Command example: ./console tests:sync-ui-screenshots 327 --repository=piwik/plugin-ActivityLog --http-user="{USERNAME}" --http-password="{PASSWORD}"

Correct Command: ./console tests:sync-ui-screenshots 327 --repository=innocraft/plugin-ActivityLog --http-user="{USERNAME_FROM_KEYPASS}" --http-password="{PASSWORDFROM_KEYPASS}"

Refer: https://github.com/matomo-org/travis-scripts/blob/master/travis.sh#L53

### PR DESCRIPTION
**Pull request by sgiehl**
Note: There are quite some more occurrences of `Piwik`, but replacing them would require to regenerate all `.travis.yml` files in all repos. Decided not to do that by now, as that might take too much time. Guess we can think about replacing the rest, when we need to update the travis files due another required change maybe.

fixes #71

### CODE REVIEW

File: travis.sh [hunk 1]
```diff
@@ -50,7 +50,7 @@ then
         echo -n "./console tests:sync-ui-screenshots $TRAVIS_BUILD_NUMBER"
         if [ -n "$PLUGIN_NAME" ]
         then
-            echo -n " --repository=piwik/plugin-$PLUGIN_NAME"
+            echo -n " --repository=matomo-org/plugin-$PLUGIN_NAME"
```
Comment (tsteur) [inline travis.sh:53]: I don't think this would work for innocraft plugins.

**Comment by tsteur** [comment]
be good indeed to not change anything in .travis.yml as we wouldn't want any extra work for that 👍

File: travis.sh (hunk 1 above)
Comment (sgiehl) [reply travis.sh:53]: that won't work for any plugin not in matomo-org scope, but guess that never worked before. Will replace it with `$TRAVIS_REPO_SLUG`, guess that should work.

**Comment by tsteur** [comment]
Merged. Does it mean we need to update the submodule in Matomo core?

**Comment by tsteur** [comment]
or its covered by the weekly auto update?

**Comment by sgiehl** [comment]
Should be covered by the auto update. But you can also trigger the action manually if the submodule update should be needed earlier.

**Pull request merged**